lib/api.js

```diff
@@ -67,14 +67,7 @@ export function ecrecover(hash, sig, v, malleabilityFlag) {
     }
     return env.read_register(0);
 }
-export function panic(msg) {
-    if (msg !== undefined) {
-        env.panic(msg);
-    }
-    else {
-        env.panic();
-    }
-}
+// NOTE: "env.panic(msg)" is not exported, use "throw Error(msg)" instead
 export function panicUtf8(msg) {
     env.panic_utf8(msg);
 }
```

Comment:
is it intentional to keep this? Or is this intended to be if someone did not want to include the debug info (line number, etc)?

Reply:
See our discussion here: https://github.com/near/near-sdk-js/issues/131#issuecomment-1208869788